usaspending_api/download/filestreaming/download_generation.py

```diff
@@ -430,7 +430,15 @@ def apply_annotations_to_sql(raw_query, aliases):
         f'{deriv_dict[alias] if alias in deriv_dict else selects_list.pop(0)} AS "{alias}"' for alias in aliases
     ]
 
-    return raw_query.replace(query_before_from, ", ".join(values_list), 1)
+    sql = raw_query.replace(query_before_from, ", ".join(values_list), 1)
+
+    # Now that we've converted the queryset to SQL, cleaned up aliasing for non-annotated fields, and sorted
+    # the fields, there's one final step.  For annotated fields, naming conflicts occur at the time they are
+    # applied to the queryset which means they never get to this function.  To work around this, we give them
+    # a name that cannot conflict with a field name on the model and then remove that conflict resolver here.
+    # As of this writing, the conflict resolver is "_NAME_CONFLICT_RESOLVER" appended to the conflicting name.
+    # Let's remove that.
+    return sql.replace("_NAME_CONFLICT_RESOLVER", "")
```

Comment:
I love this, and I want every future developer to understand how glorious this is. I suggest we put this as a global constant at the top of this file, and change all the columns using it to reference that constant.

Reply:
I felt utterly filthy when I wrote this.  I'm still pretty ashamed of it, but after having slept on it I've come to a realization; of all the solutions I tried and all the other suggestions I received to solve this issue, this one only took a one line code change and is trivial to understand.

Anyhow, I'm sure you're being sarcastic, but I feel a lot better about this solution today.

Reply:
I see my humor failed me today. I honestly enjoy how silly it is, and don't think it's a bad solution at all. However, it is a magic string, and as such I will not approve this PR until is has been replaced with a proper constant and referenced everywhere else it is used.

Reply:
Oh @BrianZito.  I don't think you realize what you've done.  That one little constant?  That has caused a circular import.  While attempting to resolve that, I decided to also reconcile our two flavors of the `apply_annotations_to_sql`... which caused me to write another unit test... which caused me to discover a bug in `apply_annotations_to_sql` (one of I'm sure hundreds)... I'll probably have rewritten USAspending by the end of the day.

Reply:
`Not only so, but we also glory in our sufferings, because we know that suffering produces perseverance; perseverance, character; and character, hope.` 

`What doesn't kill me, makes me stronger`

`The more you sweat the less you bleed` 

`Get er done!`

If you need any kind of assistance I'm happy to help.